test/spec/repeat.spec.js

```diff
@@ -710,7 +725,581 @@ describe('repeat functionality', () => {
             form.init();
             // Issue where a calculation inside a repeat is cached before the repeats are initialized (which removes the first repeat, before adding it)
             // This results in two cached calculations (for the same node) of which one no longer exists.
-            expect(form.getRelatedNodes('data-calculate').length).to.equal(1);
+            expect(
+                form.getRelatedNodes('data-calculate', '[name="/data/rg/row"]')
```

Comment:
What is this change for?

Reply:
Another leftover from changes extracted into new forms 🙃